examples/PythonProvider/python-django-webapp-sample/quickstartproject/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for quickstartproject project.
+
+Generated by 'django-admin startproject' using Django 4.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+import os
+from pathlib import Path
+from azure.appconfiguration.provider import load_provider, AzureAppConfigurationKeyVaultOptions
+from azure.identity import DefaultAzureCredential
+
+ENDPOINT = os.environ.get("AZURE_APPCONFIG_ENDPOINT")
+credential = DefaultAzureCredential()
+keyvault_options = AzureAppConfigurationKeyVaultOptions(credential=credential)
+config = load_provider(endpoint=ENDPOINT, key_vault_options=keyvault_options, credential=credential)
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = config.get('secret_key')
```

Comment:
I suppose the `secret_key` is a key vault reference. It will be good to explain this somewhere.